monai/data/dataset.py

```diff
@@ -173,7 +173,7 @@ def __init__(
             if not self.cache_dir.is_dir():
                 raise ValueError("cache_dir must be a directory.")
 
-    def _pre_transform(self, item_transformed):
+    def _preprocessing(self, item_transformed):
```

Comment:
I think the previous usage of `pre_transform` here is correct. https://github.com/Project-MONAI/MONAI/issues/2410 is mainly about the workflows.

Reply:
Sure, restored all the `_pre_transform` and `_post_transform` in datasets in the latest commit.

Thanks.